.gitignore

```diff
@@ -123,3 +123,6 @@ eox_nelp/static/*.svg
 eox_nelp/static/*.png
 eox_nelp/static/*/*.svg
 eox_nelp/static/*/*.png
+
+src
+eox_nelp/i18n/messages/
```

Comment:
I was thinking how the translations are read. Then I realize that the built files increase 400kb of size passing from 1.2MB to 1.6 MB. The good notice is that we have like CDN for that.

All the translations are there, interesting mechanism.


![image](https://github.com/user-attachments/assets/b6a59d8a-b989-4ff3-a710-bf64385365c8)